crates/hotshot/src/traits/networking/web_sever_libp2p_fallback.rs

```diff
@@ -1,318 +0,0 @@
-//! Networking Implementation that has a primary and a fallback newtork.  If the primary
-//! Errors we will use the backup to send or receive
-use super::NetworkError;
-use crate::{
-    traits::implementations::{Libp2pNetwork, WebServerNetwork},
```

Comment:
Wait, why do we have two files differing by one character? bahaha nice catch!

Reply:
Haha, I had removed it but I think during a merge it came back